tests/Integration/OptionsTest.php

```diff
@@ -0,0 +1,175 @@
+<?php
+/**
+ * Integration Tests: Plugin options
+ *
+ * @package Parsely\Tests
+ */
+
+declare(strict_types=1);
+
+namespace Parsely\Tests\Integration;
+
+use Parsely\Parsely;
+
+/**
+ * Integration Tests for plugin options.
+ */
+final class OptionsTest extends TestCase {
+	/**
+	 * Internal variable.
+	 *
+	 * @var Parsely $parsely Holds the Parsely object.
+	 */
+	private static $parsely;
+
+	/**
+	 * Setup method called before each test.
+	 */
+	public function set_up(): void {
+		parent::set_up();
+
+		self::$parsely = new Parsely();
+	}
+
+	/**
+	 * Verifies that get_options() can handle corrupted or unset values.
+	 *
+	 * @since 3.0.0
+	 *
+	 * @covers \Parsely\Parsely::get_options
+	 */
+	public function test_default_options_are_returned_when_options_are_corrupted_or_not_set(): void {
+		update_option( Parsely::OPTIONS_KEY, 'someinvalidvalue' );
+
+		$options = self::$parsely->get_options();
+		self::assertSame( self::$parsely->get_default_options(), $options );
+
+		delete_option( Parsely::OPTIONS_KEY );
+		self::assertSame( self::$parsely->get_default_options(), $options );
+	}
+
+	/**
+	 * Verifies that get_options() returns the correct "Track As" defaults.
+	 *
+	 * @since 3.9.0
+	 *
+	 * @covers Parsely\Parsely::get_options
+	 */
+	public function test_get_options_returns_correct_track_as_defaults(): void {
+		$options = self::$parsely->get_options();
+		self::assertSame( array( 'post' ), $options['track_post_types'] );
+		self::assertSame( array( 'page' ), $options['track_page_types'] );
+
+		delete_option( Parsely::OPTIONS_KEY );
```

Comment:
Same as above

Reply:
Right. So what you're saying if I understand correctly is that since we have never set the option, it will be already absent from the database, thus deletion makes no sense.